### DISCUSSION TITLE
For the output result of CVHD with colvars module , how to obtain the value of global collective variable（cv） η according to the paper，whose title is 'Merging metadynamics into hyperdynamics: accelerated molecular simulations Reaching time scales from microseconds to seconds'

### DISCUSSION THREAD

**Discussion by 12jscvb** [Colvars/colvars#663] [Q&A]
Hello!  everyone , I am new to CVHD. I encounter some difficult problems, may I ask you for advice? (The attachment is the out.colvars.traj file)

The out.colvars.traj file obtained by lammps running the CVHD method through the modified colvars module, the first line of the content is 'step', 'coord', and 'E_hd'. I originally thought that the 'coord' parameter represented the global collective variable(cv) η mentioned in the paper, but I found that its value gradually increased from  e-01 to e+02 . What is the explanation of the coord parameter? How does this parameter relate to the global collective variable η mentioned in the paper? (or how to get the η value) 

 Thank you very much for help!
[1.txt](https://github.com/Colvars/colvars/files/14392944/1.txt)

**Comment by jhenin** [comment]
Hi! This is an external patch to Colvars, written, as far as I know, by Kristof Bal (@kbal). We have nothing to do with it.
For context for other readers, the paper is: https://pubs.acs.org/doi/abs/10.1021/acs.jctc.5b00597 - which includes source files for CVHD/Colvars.
We (the Colvars developers) are not in a position to help you, sorry. Maybe the author will be.

**Comment by giacomofiorin** [comment]
Hi @12jscvb, I assume that [this](https://doi.org/10.1021/acs.jctc.5b00597) is the paper you are referring.

We document all builtin functions that are used to define collective variables, but in your case we do not know the parameters of the function used, nor which one. You also refer to a "modified" Colvars module, in which case we can help even less.

**Comment by 12jscvb** [comment]
ok,Thank you for suggestions. Also, For the' COLLECTIVE VARIABLES MODULE After the Reference manual for LAMMPS', I have several questions to ask the professor for advice.
1 In  the 'Defining grid parameters for a colvar' section, I have diffcult in understanding the meaning of the 'width' parameter. And how to set the value of this parameter as a general rule of thumb
2 How to set the 'hillweight' parameter in the Required metadynamics keywords section?  
Thanks!

**Comment by giacomofiorin** [comment]
Hi @12jscvb 
> ok,Thank you for suggestions. Also, For the' COLLECTIVE VARIABLES MODULE After the Reference manual for LAMMPS', I have several questions to ask the professor for advice. 1 In the 'Defining grid parameters for a colvar' section, I have diffcult in understanding the meaning of the 'width' parameter. 

The main usages of [`width`](https://colvars.github.io/master/colvars-refman-lammps.html#colvar|width) are to (1) set the default value of the grid spacing and (2) set a unit of length so that you can [express the force constant in energy units](https://colvars.github.io/master/colvars-refman-lammps.html#sec:colvarbias_harmonic). Unless you have a reason to set a value for `width` based on these cases, leave it as default.

> And how to set the value of this parameter as a general rule of thumb 2 How to set the 'hillweight' parameter in the Required metadynamics keywords section? Thanks!

The lowest value of `hillWeigth` the better, provided that you can still fill up the landscape in the affordable simulation time. How good that compromise is is entirely dependent on the specific application.
